benchmarks/nemo/nemolite2d/manual_versions/psykal_opencl/allkernels_tasks.cl

```diff
@@ -0,0 +1,482 @@
+
+// LEN value predefined for a problem size of 250x250
```

Comment:
Please mention this restriction in the README.

Reply:
Done